app/views/discovered_hosts/multiple_destroy.html.erb

```diff
@@ -1,6 +1,8 @@
 <%= form_tag submit_multiple_destroy_discovered_hosts_path(:host_ids => params[:host_ids]), :onsubmit => "resetSelection()" do -%>
-  <span class="label label-warning"><%= _('Warning') -%></span>
-  <%= _('This might take a while, as all hosts, facts and reports will be destroyed as well') %>
-<% end %><br />
+  <div class="alert alert-warning">
```

Comment:
please use the alert helper instead (https://github.com/theforeman/foreman/blob/develop/app/helpers/layout_helper.rb#L96)